packages/nitro-rpc-client/src/interface.ts

```diff
@@ -100,11 +101,25 @@ interface paymentApi {
 
 interface syncAPI {
   /**
-   * WaitForObjective blocks until the objective with the given ID to complete.
+   * WaitForLedgerChannelToHaveStatus blocks until the ledger channel with the given ID to have the given status.
    *
-   * @param objectiveId - The id objective to wait for
+   * @param objectiveId - The channel id to wait for
+   * @param status - The channel id to wait for (e.g. Ready or Closing)
+   */
+  WaitForLedgerChannelToHaveStatus(
```

Comment:
Nit: could change the function name to `WaitForLedgerChannelStatus` to be a bit more concise

Reply:
[6e06584](https://github.com/statechannels/go-nitro/pull/1829/commits/6e0658447119880bdc2243e0251e422d8a408113)